widgets/flood/description.js

```diff
@@ -0,0 +1,34 @@
+var R = require('react');
+var DataTable = require('../dataTable');
+var D = R.DOM;
+
+var Description = R.createClass({
+  displayName: 'flood-description',
+  propTypes: {
+    floodData: R.PropTypes.object,
+    isAtRisk: R.PropTypes.bool
+  },
+
+  render: function() {
+    var fields = [
+      ['dataPublishedOn', 'Data published'],
+      ['likelihood.riskCategory', 'Risk category'],
+      ['suitability.scale', 'Suitability scale']
```

Comment:
Wouldn't `suitability.note` be more appropriate? `scale` is not really meant to be human-readable, e.g. `TownToStreet`